wagtail/images/templates/wagtailimages/images/url_generator.html

```diff
@@ -1,7 +1,10 @@
 {% extends "wagtailadmin/base.html" %}
 {% load wagtailimages_tags wagtailadmin_tags i18n %}
 
-{% block titletag %}{% blocktrans trimmed with title=image.title %}Editing image {{ title }}{% endblocktrans %}{% endblock %}
+{% block titletag %}
+    {% blocktrans trimmed with title=image.title %}Editing image
+        {{ title }}{% endblocktrans %}
+{% endblock %}
```

Comment:
Please revert this unintended formatting, it could be the way Prettier is running on your laptop maybe?

Reply:
Likely so. Sorry about that. I'll revert.